driver/SevenSegment.h

```diff
@@ -0,0 +1,45 @@
+#ifndef SEVENSEGMENTHEADER_H  
+#define SEVENSEGMENTHEADER_H  
+
+#include "stm32xx_hal.h"
+
+#define NUMBER_SEVEN_SEGMENTS 2
+#define NUMBER_SEGMENTS 7
+
+struct led_pins { 
+    uint32_t Pin;
+    GPIO_TypeDef *Port;
+}; 
+
+GPIO_InitTypeDef led_configA = {
+    .Mode = GPIO_MODE_OUTPUT_PP,
+    .Pull = GPIO_NOPULL,
+    .Pin = 0 
+}; 
+GPIO_InitTypeDef led_configB = {
+    .Mode = GPIO_MODE_OUTPUT_PP,
+    .Pull = GPIO_NOPULL,
+    .Pin = 0
+}; 
+
+GPIO_InitTypeDef led_configC = {
+    .Mode = GPIO_MODE_OUTPUT_PP,
+    .Pull = GPIO_NOPULL,
+    .Pin = 0
+}; 
```

Comment:
Are these configs going to be used/changed in other files? If not, we can just limit the scope to the `.c` file and make them static in `.c`.